Example/Tests/Tests.swift

```diff
@@ -58,6 +58,7 @@ class Tests: XCTestCase {
 		XCTAssertEqual(slider.trackShadowDepth, 0)
 		XCTAssertEqual(slider.trackMinAngle, 0)
 		XCTAssertEqual(slider.trackMaxAngle, 360)
+        XCTAssertEqual(slider.trackRounding, false)
```

Comment:
Indentation's off? I think it's because the file is using tabs but you're using spaces.

Reply:
This is strange because I am using the auto-formatting tool from Xcode. But I'll fix it anyway.